third_party/crashpad/wrapper/wrapper.cc

```diff
@@ -211,24 +210,13 @@ void InstallCrashpadHandler(bool start_at_crash,
   InitializeCrashpadDatabase(database_directory_path);
   client->SetUnhandledSignals({});
 
-  if (start_at_crash)
-    client->StartHandlerAtCrash(handler_path,
+  client->StartHandlerAtCrash(handler_path,
                                 database_directory_path,
                                 default_metrics_dir,
                                 kUploadUrl,
                                 ca_certificates_path,
                                 default_annotations,
                                 default_arguments);
-  else
-    client->StartHandler(handler_path,
-                         database_directory_path,
-                         default_metrics_dir,
-                         kUploadUrl,
-                         ca_certificates_path,
-                         default_annotations,
-                         default_arguments,
-                         false,
-                         false);
```

Comment:
It would be good to also clean up the customizations we've made to this API and its implementation in the Crashpad code (for example, to support b/201538792). That doesn't need to happen in this PR, but we should do it soon. Let me know if you'd rather me handle that piece.

Reply:
Yep, will look into it later.